### DISCUSSION TITLE
Loading custom .yaml config for checkpoint?

### DISCUSSION THREAD

**Discussion by anon0730** [lllyasviel/stable-diffusion-webui-forge#77] [Q&A]
I use an SD1.5 model that requires a custom .yaml file to work properly because it uses parameterization: "v" like SD2 models do. 
In the original webui I simply put the yaml in the same folder as the checkpoint with the same filename and it gets loaded automatically. Is there a way to load this yaml with the model in Forge?
The difference to v1-inference.yaml is rather small overall, just two new lines but without it the outputs are broken.
![vfeEkqFC 1](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/132453170/1b9d9791-1c92-4542-b82e-a86fc0612e5d)

**Comment by bendecruz745** [comment]
At the moment it seems like load_model in the sd_models script does not account for/read config files (likely incomplete? Comparing to A1111 setup of this file)

If you wanted a rough workaround , update the line in the model_base which currently checks if it is an EPS model and instead replace the return 'c' value to V_PREDICTION, this allowed me to get VPRED models to load and function.
https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/b58b0bd4259cf71077dfd7787fb77af4c02760a1/ldm_patched/modules/model_base.py#L24

Note though that this does break all other models that do not use it, so you will have to change this back if so, which may be annoying depending on how often you change models.

To add on: there is likely a way to set the Model_Type variable fetched from the checkpoint directly (likely via a merger) but I have no idea how to do that.

**Reply by mkgiga** [reply]
Before the major refactor and recent changes to this repository I used to use this as a workaround. Does anyone know how/where forge loads custom checkpoint configs (if this is implemented yet, I mean) because vpred models are still broken

**Comment by anon0730** [comment]
The workaround helps a lot, thanks! Hopefully there will be support for config files in time.
